ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthEstimateGas.java

```diff
@@ -91,6 +91,8 @@ private JsonCallParameter overrideGasLimitAndPrice(
         callParams.getTo() != null ? callParams.getTo().toString() : null,
         Quantity.create(gasLimit),
         Quantity.create(0L),
+        callParams.getGasPremium().map(Quantity::create).orElse(null),
+        callParams.getFeeCap().map(Quantity::create).orElse(null),
```

Comment:
I think while we're here we might as well fix this issue of `CallParams` and `JsonCallParams`. Take a look at `FilterParameter`, that has non-string types but also has the properties to make it json serializable. If you want to make it a separate PR that works!

Reply:
I will do this in another PR because there are already two things in this PR with `eth_call` and `eth_estimateGas`